src/main.js

```diff
@@ -109,9 +109,9 @@ class MiniMediaPlayer extends LitElement {
       toggle_power: true,
       ...config,
       hide: { ...DEFAULT_HIDE, ...config.hide },
-      sonos: {
+      speaker_group: {
         show_group_count: true,
```

Comment:
```js
platform: 'sonos',
```
default to sonos to avoid breaking change

Reply:
Added in new commit